calibration/CTDAVN/CGINS-CTDAVN-00424__20171006.csv

```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+424,CC_calibration_date,1507248000000,sensor calibration date (milliseconds since 1970-01-01)

```

Comment:
Renamed to resolve @rtravis-whoi comment about FN. I think her comments went away since I renamed it via the browser

Reply:
looks good